manifests/modelserving/servingruntimes-config.yaml

```diff
@@ -0,0 +1,55 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: servingruntimes-config
+data:
+  default-config: |
```

Comment:
Is there a reason we are providing this YAML? Isn't the whole point to fall back on ours and then if they WANT to, they can override it? Is this some sorta self documenting YAML?

I ask this because you're duplicating efforts here -- any change to the default needs to be done to this -- and they are not tied together.

Reply:
This yaml is only for upstream and it's not even included in the manifests, in downstream will be handle by the deployer.

Reply:
I stand by thinking this is a DRY issue -- but if you think it's needed, we can keep it